packages/addons/addon-depends/protobuf/package.mk

```diff
@@ -2,11 +2,11 @@
 # Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="protobuf"
-PKG_VERSION="3.20.3"
-PKG_SHA256="e51cc8fc496f893e2a48beb417730ab6cbcb251142ad8b2cd1951faa5c76fe3d"
+PKG_VERSION="21.12"
```

Comment:
is there a reason we switch the version ?
likewise they play version bingo at protobuf but should we stay at the current versioning ?
they have 3 different versions with the same hash for some reason
isn`t 24.2 the latest version and everything else is "something else" ? 

![image](https://github.com/LibreELEC/LibreELEC.tv/assets/1355173/58c242fc-ef1c-418d-beb9-6e88cf6b57d0)

Reply:
21 is actually 3 based on my navigation. Which is why I went with this. Also jumping to 4 or 22 - changes the packaging - and as you say is not the same product.